.github/workflows/playwright.yml

```diff
@@ -0,0 +1,60 @@
+name: Lint, typecheck, build, run and e2e test the CRGA
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Checkout sources
+      uses: actions/checkout@v3
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16
+
+    - name: Install dependencies
+      run: npm ci
+
+    - name: Install Playwright Browsers
+      run: npx playwright install --with-deps
```

Comment:
We probably want to move the playwright installation after the successful `npm run build` step since `test` and `build` might fail.

Reply:
https://github.com/terrestris/create-react-geo-app/pull/7/commits/d4834706d9b07e71be36822a83a03af3cfbeba5c